mod-permissions/src/main/resources/domain/mod-permissions/features/permissions.feature

```diff
@@ -0,0 +1,398 @@
+Feature: Permissions tests
+
+  Background:
+    * url baseUrl
+    * call login testAdmin
+    * def commonHeaders =
+    """
+    {
+      "x-okapi-tenant": "#(testTenant)",
+      "x-okapi-token": "#(okapitoken)",
+      "Content-Type": "application/json",
+      "Accept": "application/json, text/plain"
+    }
+    """
+    * def optionsHeaders =
+    """
+    {
+      "Access-Control-Allow-Headers": "Content-Type, X-Okapi-Tenant, X-Okapi-Token, Authorization, X-Okapi-Request-Id, X-Okapi-Module-Id",
+      "Access-Control-Request-Method": "PUT, PATCH, DELETE, GET, POST",
+      "Access-Control-Allow-Origin": "*",
+      "Origin": "#(baseUrl)"
+    }
+    """
+    * configure lowerCaseResponseHeaders = true
+    * def testUserId = "00000000-1111-5555-9999-999999999998"
+    * def permissionsUserId = "00000000-1111-5555-9999-888888888885"
```

Comment:
Can those be generated or has to be static?

Reply:
Agree this doesn't seem right. I got stuck because doing `*def id = uuid()` was generating a new uuid everytime the variable was used (it was a function reference in other words rather than the value). But if you do `* def  id = callonce uuid` things work as expected. I'll make the change.